src/test/java/com/hyperwallet/clientsdk/model/HyperwalletPaymentTest.java

```diff
@@ -10,14 +10,13 @@ protected HyperwalletPayment createBaseModel() {
         HyperwalletPayment payment = new HyperwalletPayment();
         payment
                 .status("COMPLETED")
+                .transition("COMPLETED")
```

Comment:
CANCELLED is the only available transition for HyperwalletPayment.

Reply:
Following are the status transitions allowed for HyperwalletPayment. 'COMPLETED' is one of them and 'CANCELLED' is not one of them. Details are per https://docs.hyperwallet.com/content/api/v3/resources/status-transitions/list. Please let me know if there is any error in my statement.

CREATED
SCHEDULED
PENDING_ACCOUNT_ACTIVATION
PENDING_ID_VERIFICATION
PENDING_TAX_VERIFICATION
PENDING_TRANSFER_METHOD_ACTION
PENDING_TRANSACTION_VERIFICATION
IN_PROGRESS
UNCLAIMED
COMPLETED
FAILED
RECALLED
RETURNED
EXPIRED